3-box-model/style.css

```diff
@@ -0,0 +1,29 @@
+footer {
```

Comment:
```suggestion
.footer {
```
Prefer classnames over tag names so your markup doesn't have to match your styles.